http/server.go

```diff
@@ -91,11 +91,10 @@ func (server *APIServer) Start(edgeMode bool) error {
 		WriteTimeout: 30 * time.Minute,
 	}
 
-	log.Printf("[INFO] [http] [server_addr: %s] [server_port: %s] [secured: %t] [api_version: %s] [message: Starting Agent API server]", server.addr, server.port, config.Secured, agent.Version)
+	log.Printf("[INFO] [http] [server_addr: %s] [server_port: %s] [use_tls: %t] [api_version: %s] [message: Starting Agent API server]", server.addr, server.port, config.UseTLS, agent.Version)
 
-	if edgeMode {
+	if !config.UseTLS {
```

Comment:
I would have kept that condition here as the code in the condition is actually related to Edge mode and not if the server is exposed over HTTP or HTTPS.

Reply:
ok, in that case.